cpp/include/raft/stats/trustworthiness_score.cuh

```diff
@@ -48,6 +50,42 @@ double trustworthiness_score(const raft::handle_t& h,
   return detail::trustworthiness_score<math_t, distance_type>(
     h, X, X_embedded, n, m, d, n_neighbors, batchSize);
 }
+
+/**
+ * @brief Compute the trustworthiness score
+ * @tparam value_t the data type
+ * @tparam idx_t Integer type used to for addressing
+ * @param[in] handle the raft handle
+ * @param[in] X: Data in original dimension
+ * @param[in] X_embedded: Data in target dimension (embedding)
+ * @param[in] n_neighbors Number of neighbors considered by trustworthiness score
+ * @param[in] batch_size Batch size
+ * @return Trustworthiness score
+ * @note The constness of the data in X_embedded is currently casted away and the data is slightly
+ * modified.
+ * TODO: Change the underlying implementation to remove the need to const_cast X_embedded.
```

Comment:
Same as above, move the todo inside the implementation. We want the docs to be strictly for users